cmd/cli/kubectl-kyverno/commands/apply/command.go

```diff
@@ -590,6 +605,60 @@ func (c *ApplyCommandConfig) applyImageValidatingPolicies(
 	return responses, nil
 }
 
+// Adding maps
+func (c *ApplyCommandConfig) applyMutatingAdmissionPolicies(
```

Comment:
This function should be removed since the mutation logic is added in the policy processor.